name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DADATBankenhausPDFExtractor.java

```diff
@@ -298,6 +298,11 @@ private void addDividendeTransaction()
                     BigDecimal exchangeRate = asExchangeRate(v.get("exchangeRate"));
                     type.getCurrentContext().put("exchangeRate", exchangeRate.toPlainString());
                 })
+                
+                // Ex-Dividenden-Tag 4.3.2021
```

Comment:
// Ex-Dividenden-Tag 4.3.2021
not the same for regEx
`^Extag: (?<exDate>\\d+.\\d+.\\d{4})$`

Reply:
changed